Civility Optics/Networking/NetworkingService.swift

```diff
@@ -414,6 +417,72 @@ extension NetworkingService {
       }.resume()
     }
     
+    static func report(
+        id: String
+    ) {
+      var req = URLRequest(url: URL(string: baseURL + "ratings/flag")!)
+      req.httpMethod = "POST"
+      req.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        let body: [String : String] =  ["_id": id]
+      req.httpBody = try? JSONEncoder().encode(body)
+      
+      URLSession.shared.dataTask(with: req) { data, res, error in
+        guard
+          let data = data,
+          let res = res as? HTTPURLResponse,
+          error == nil
+        else {
+          print("Error", error ?? "Unknown error")
+          return
+        }
+        
+        guard checkStatus(res) else {
+          return
+        }
+        
+        printResponse(data)
+      }.resume()
+    }
+    
+    
+    static func helpful(
+        email: String,
+        id: String
+    ) {
+      var req = URLRequest(url: URL(string: baseURL + "ratings/helpful")!)
+      req.httpMethod = "POST"
+      req.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        let body: [String : AnyEncodable] =  [
+            "email": AnyEncodable(email),
+            "_id": AnyEncodable(id),
+        ]
+      req.httpBody = try? JSONEncoder().encode(body)
+      
+      URLSession.shared.dataTask(with: req) { data, res, error in
+        guard
+          let data = data,
+          let res = res as? HTTPURLResponse,
+          error == nil
+        else {
+          print("Error", error ?? "Unknown error")
+          return
+        }
+        
+        guard checkStatus(res) else {
+          return
+        }
+        
+        printResponse(data)
+      }.resume()
+    }
+    
+    
+    
```

Comment:
We need to handle merge conflicts with frontend auth, so that our routes use auth.